src/webui/abstractwebapplication.cpp

```diff
@@ -234,14 +234,15 @@ QString AbstractWebApplication::generateSid()
 
 void AbstractWebApplication::translateDocument(QString& data)
 {
-    const QRegExp regex("QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR");
+    const QRegExp regex("QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR(\\(CONTEXT=([a-zA-Z_][a-zA-Z0-9_]*)([\\)]))?");
```

Comment:
Maybe it would be better to move translation parameters definition inside QBT_TR macro? Example:
QBT_TR(Some text[CONTEXT=ContextName])QBT_TR